samples/ObjCDemoApp/Podfile

```diff
@@ -1,8 +1,8 @@
-source 'https://github.com/CocoaPods/Specs.git'
-platform :ios, '11.0'
+source 'https://cdn.cocoapods.org/'
```

Comment:
Using `https://github.com/CocoaPods/Specs.git` is not recommended:
https://blog.cocoapods.org/CocoaPods-1.8.0-beta/
(though I hope the CI is using a recent enough version of CocoaPods)